nerfstudio/configs/external_methods.py

```diff
@@ -0,0 +1,160 @@
+# Copyright 2022 the Regents of the University of California, Nerfstudio Team and contributors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# pylint: disable=invalid-name
+
+"""This file contains the configuration for external methods which are not included in this repository."""
+import subprocess
+import sys
+from dataclasses import dataclass, field
+from typing import Any, Dict, Optional, Tuple, cast
+
+from rich.prompt import Confirm
+
+from nerfstudio.engine.trainer import TrainerConfig
+from nerfstudio.utils.rich_utils import CONSOLE
+
+
+@dataclass
+class ExternalMethod:
+    """External method class. Represents a link to a nerfstudio-compatible method not included in this repository."""
+
+    slug: str
+    model_description: str
+    instructions: str
+    install_commands: Optional[str] = None
+
+
+pip = f"{sys.executable} -m pip"
+external_methods = []
+
+# Instruct-NeRF2NeRF
+in2n_install_instructions = """
+[bold yellow]Instruct-NeRF2NeRF[/bold yellow]
+For more information visit: https://docs.nerf.studio/en/latest/nerfology/methods/in2n.html
+
+To enable Instruct-NeRF2NeRF, you must install it first by running:
+  [grey]pip install git+https://github.com/ayaanzhaque/instruct-nerf2nerf[/grey]
+""".strip()
+in2n_install_script = f"""{pip} install git+https://github.com/ayaanzhaque/instruct-nerf2nerf"""
+external_methods.extend(
+    [
+        ExternalMethod(
+            "in2n", "Instruct-NeRF2NeRF. Full model, used in paper", in2n_install_instructions, in2n_install_script
+        ),
+        ExternalMethod(
+            "in2n-small", "Instruct-NeRF2NeRF. Half precision model", in2n_install_instructions, in2n_install_script
+        ),
+        ExternalMethod(
+            "in2n-tiny",
+            "Instruct-NeRF2NeRF. Half prevision with no LPIPS",
+            in2n_install_instructions,
+            in2n_install_script,
+        ),
+        ExternalMethod(
+            "in2n-tiny",
+            "Instruct-NeRF2NeRF. Half prevision with no LPIPS",
+            in2n_install_instructions,
+            in2n_install_script,
+        ),
+    ]
+)
```

Comment:
Not sure if it is any better, but what about something more concise, like

```
# Instruct-NeRF2NeRF
in2n_install_instructions = """
[bold yellow]Instruct-NeRF2NeRF[/bold yellow]
For more information visit: https://docs.nerf.studio/en/latest/nerfology/methods/in2n.html
To enable Instruct-NeRF2NeRF, you must install it first by running:
  [grey]pip install git+https://github.com/ayaanzhaque/instruct-nerf2nerf[/grey]
""".strip()
external_methods.append(
       ExternalMethod(
            methods={
                   "in2n": "Instruct-NeRF2NeRF. Full model, used in paper",
                   "in2n-small": "Instruct-NeRF2NeRF. Half precision model", 
                   "in2n-tiny": "Instruct-NeRF2NeRF. Half prevision with no LPIPS", 
             }
             instructions: in2n_install_instructions,
             install_repo: "https://github.com/ayaanzhaque/instruct-nerf2nerf"
)
```

Reply:
Is it better now?

Reply:
Yea, I like it more.